flair/datasets/sequence_labeling.py

```diff
@@ -4839,6 +4839,137 @@ def __init__(
         )
 
 
+class NOISY_NER_EST(ColumnCorpus):
+    data_url = "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/patnlp/estner.cnll.zip"
+    label_url = "https://raw.githubusercontent.com/uds-lsv/NoisyNER/master/data/only_labels"
+
+    def __init__(
+        self,
+        version: int = 0,
+        base_path: Optional[Union[str, Path]] = None,
+        in_memory: bool = True,
+        **corpusargs,
+    ) -> None:
+        """Initialize the NoisyNER corpus.
+
+        :param version: Chooses the labelset for the data.
```

Comment:
please use the [google docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) format as described in our [docs](https://flairnlp.github.io/flair/v0.13.1/contributing/updating_documentation.html#docstrings)

Reply:
Hi. I have made all the requested changes. Is the description in the correct format now?